### PR TITLE
[PUB-3000] Replace state.user references by orgs

### PR DESCRIPTION
**Pull request by anafilipadealmeida**
## Description
- Replace state.user and user_${dataFetchActionTypes.FETCH_SUCCESS} references by orgs
- Remove some orgs data from user parser and user rpc

What's left for another PR: do it for the composer, thirdParty, modals and trial, so I can clean the rest of the user parser

## Context & Notes
https://buffer.atlassian.net/browse/PUB-3000

## Checklist

-   [x] My code follows the code style and guidelines of this project. 
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have tested different user stories. 
-   [x] I have identified similar or related features and tested they are still working.
-   [x] I have performed a self-review of my own code.
-   [ ] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
-   [x] I have commented my code, particularly in hard-to-understand areas.
-   [ ] I kept accessibility in mind by following the [a11y checklist.](https://www.notion.so/buffer/Workflow-Checklist-e64d86eb795140bcbfdc16d1c72e573f)
-   [ ] I have considered [security, abuse & compliance](https://www.notion.so/buffer/Engineering-Wiki-f34142d290304c35bebadf76cc9cc89e#cc6dcc7617184227b77da2e1b262a563) implications of my changes.

#### Staging Deployment

To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/modals/middleware.js [hunk 1]
```diff
@@ -40,10 +40,6 @@ export default ({ dispatch, getState }) => next => action => {
       }
       break;
     case 'INIT_MODALS': {
-      if (getState().user.plan === 'awesome') {
```
Comment (albennett) [inline packages/modals/middleware.js:43]: 💅

File: packages/profile-sidebar/reducer.js [hunk 2]
```diff
@@ -51,7 +51,12 @@ const moveProfileInArray = (arr, from, to) => {
   return clone;
 };
 
-const handleProfileDropped = (profiles, action, userId, isFreeUser) => {
+const handleProfileDropped = (
```
Comment (albennett) [inline packages/profile-sidebar/reducer.js:54]: Not a blocker, but these arguments could be a good one to refactor with destructuring since there's four.

**Review by albennett — APPROVED** [review]
LGTM 🚀 There's so much to refactor with the org clean up. You're doing great 🙌

File: packages/profile-sidebar/reducer.js (hunk 2 above)
Comment (anafilipadealmeida) [reply packages/profile-sidebar/reducer.js:54]: Definitely! At this point, with so much still to clean I can't give proper attention to some of these things, but going through so many files is giving us a better idea of what places need some more love 😅

**Pull request merged**